staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/validation_test.go

```diff
@@ -1353,6 +1353,171 @@ func TestValidationExpressions(t *testing.T) {
 				// TODO: also find a way to test the errors returned for: array with no items, object with no properties or additionalProperties, invalid listType and invalid type.
 			},
 		},
+		{name: "stdlib list functions",
+			obj: map[string]interface{}{
+				"ints":         []interface{}{int64(1), int64(2), int64(2), int64(3)},
+				"unsortedInts": []interface{}{int64(2), int64(1)},
+				"emptyInts":    []interface{}{},
+
+				"doubles":         []interface{}{float64(1), float64(2), float64(2), float64(3)},
+				"unsortedDoubles": []interface{}{float64(2), float64(1)},
+				"emptyDoubles":    []interface{}{},
```

Comment:
I can't remember if we already dealt with this, but a double expressed in json with no floating point gets decoded as int64s

since cel is looking at schema info, it'll be expecting doubles... does the conversion on standalone and list values work properly so a cel expression like `self.doubles.min() == 1.0` evaluates properly against a request input like `{"doubles":[1,1.5,2,2.5]}`?

Reply:
I remember you caught the need to handle this when reviewing the [wrapper code](https://github.com/kubernetes/kubernetes/blob/3213a92802b91ed718578f51b14916105fb0c3da/staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/values.go#L170).  I'll add some "int backed doubles" to this test as well.